test/OrchardCore.Tests/Modules/OrchardCore.OpenId/OpenIdApplicationStepTests.cs

```diff
@@ -0,0 +1,186 @@
+using System;
+using System.Collections.Immutable;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Extensions.FileProviders;
+using Moq;
+using Newtonsoft.Json.Linq;
+using OpenIddict.Abstractions;
+using OrchardCore.OpenId.Abstractions.Descriptors;
+using OrchardCore.OpenId.Abstractions.Managers;
+using OrchardCore.OpenId.Recipes;
+using OrchardCore.OpenId.YesSql.Models;
+using OrchardCore.Recipes.Models;
+using OrchardCore.Tests.Utilities;
+using Xunit;
+using static OpenIddict.Abstractions.OpenIddictConstants;
+
+namespace OrchardCore.Tests.Modules.OrchardCore.OpenId
+{
+    public class OpenIdApplicationStepTests
+    {
+        private string GetRecipeFileContent(string recipeName)
```

Comment:
You can move `private` method before ending the class

Reply:
Done